src/test/java/com/neo4j/docker/TestConfSettings.java

```diff
@@ -135,4 +136,53 @@ void testReadTheConfFile() throws Exception
         //Kill the container
         container.stop();
     }
+
+    @Test
+    void testCommentedConfigsAreReplacedByDefaultOnes() throws Exception
+    {
+        //Create container
+        GenericContainer container = createContainer();
+        //Mount /conf
+        Path confMount = HostFileSystemOperations.createTempFolderAndMountAsVolume( container, "conf-", "/conf" );
+        SetContainerUser.currentlyRunningUser( container );
+        //Create neo4j.conf file
+        Path confFile = Paths.get( "src", "test", "resources", "neo4j.conf" );
+        Files.copy(confFile, confMount.resolve("neo4j.conf"));
+        //Start the container
+        container.setWaitStrategy( Wait.forLogMessage( ".*Config Dumped*." , 1 ).withStartupTimeout( Duration.ofSeconds( 10 ) ) );
+        container.setCommand("dump-config");
+        container.start();
+        //Read the config file to check if the config is appended correctly
+        Stream<String> lines = Files.lines(confMount.resolve("neo4j.conf"));
```

Comment:
The `Files.lines` code was good for parsing the `debug.log`, but there's the method `parseConfFile()` which could be used for parsing the dumped conf file. It's easier to step through with the debugger to see all the configs that are actually being set.

Reply:
this comment still applies